packages/terra-badge/package.json

```diff
@@ -31,10 +31,12 @@
     "test": "npm run test:spec && npm run test:nightwatch-default",
     "test:spec": "$(cd ..; npm bin)/jest --config ../../jestconfig.json",
     "test:all": "npm run test:nightwatch-default && npm run test:nightwatch-chrome && npm run test:nightwatch-firefox && npm run test:nightwatch-safari",
-    "test:nightwatch-default": "SPECTRE_TEST_SUITE=terra-badge node ./node_modules/terra-toolkit/lib/scripts/nightwatch.js",
-    "test:nightwatch-chrome": "SPECTRE_TEST_SUITE=terra-badge node ./node_modules/terra-toolkit/lib/scripts/nightwatch.js chrome",
-    "test:nightwatch-firefox": "SPECTRE_TEST_SUITE=terra-badge node ./node_modules/terra-toolkit/lib/scripts/nightwatch.js firefox",
-    "test:nightwatch-safari": "SPECTRE_TEST_SUITE=terra-badge node ./node_modules/terra-toolkit/lib/scripts/nightwatch-non-parallel.js safari"
+    "test:nightwatch-default": "SPECTRE_TEST_SUITE=terra-badge node ../../node_modules/terra-toolkit/lib/scripts/nightwatch.js",
+    "test:nightwatch-chrome": "SPECTRE_TEST_SUITE=terra-badge node ../../node_modules/terra-toolkit/lib/scripts/nightwatch.js chrome",
+    "test:nightwatch-firefox": "SPECTRE_TEST_SUITE=terra-badge node ../../node_modules/terra-toolkit/lib/scripts/nightwatch.js firefox",
+    "test:nightwatch-safari": "SPECTRE_TEST_SUITE=terra-badge node ../../node_modules/terra-toolkit/lib/scripts/nightwatch-non-parallel.js safari",
+    "test:remote": "SPECTRE_TEST_SUITE=terra-badge REMOTE=true node ../../node_modules/terra-toolkit/lib/scripts/nightwatch-process.js --config ../../node_modules/terra-toolkit/lib/nightwatch.conf.js --env default-tiny,default-small,default-medium,default-large,default-huge,default-enormous",
+    "test:remote:all": "SPECTRE_TEST_SUITE=terra-badge REMOTE=true node ../../node_modules/terra-toolkit/lib/scripts/nightwatch-process.js --config ../../node_modules/terra-toolkit/lib/nightwatch.conf.js --env chrome-tiny,chrome-small,chrome-medium,chrome-large,chrome-huge,chrome-enormous,firefox-tiny,firefox-small,firefox-medium,firefox-large,firefox-huge,firefox-enormous,ie10-tiny,ie10-small,ie10-medium,ie10-large,ie10-huge,ie10-enormous,ie11-tiny,ie11-small,ie11-medium,ie11-large,ie11-huge,ie11-enormous,edge-tiny,edge-small,edge-medium,edge-large,edge-huge,edge-enormous,safari-tiny,safari-small,safari-medium,safari-large,safari-huge,safari-enormous"
```

Comment:
We may want to remove the remote tests for the time being. Right now sauce labs throttles our usage so we never actually run the remote tests.